zad4/app/services/football_player_manager.rb

```diff
@@ -4,18 +4,20 @@ class FootballPlayerManager
   attr_reader :player
 
   def initialize(player)
-    @player = player
+    Player.where('power > ?', player.power)
```

Comment:
Ten `initialize` był bardzo w porządku ;) Ta metoda powinna po prostu zapisywać obiekt `Player` w zmiennej instancyjej a pozostałe metody powinny korzystac z tej zmiennej. Przez to że nic nie zapisałeś w zmiennej `@player` pozostałe metody się wykładają, spróbuj zaimplementować ten serwis jeszcze raz, ale bez ruszania implementacji initialize, a przy wykorzystaniu jej :)